package.json

```diff
@@ -13,7 +13,6 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^29.5.3",
     "@types/node": "^20.4.2",
-    "@types/react": "^18.2.15",
```

Comment:
This one either needs to be in `devDependencies` or it needs to be here. I think this was another accidental change on your part.

Reply:
I got confused with these two, I put it back in dependencies